modules/mono/glue/GodotSharp/GodotSharp/Core/Rect2.cs

```diff
@@ -74,10 +74,11 @@ public Rect2 Abs()
 
         /// <summary>
         /// Returns the intersection of this Rect2 and `b`.
+        /// If the rectangles do not intersect, an empty Rect2 is returned.
         /// </summary>
         /// <param name="b">The other rect.</param>
         /// <returns>The clipped rect.</returns>
-        public Rect2 Clip(Rect2 b)
```

Comment:
The `<returns>` comment needs to be updated too.

Reply:
Changed to `The clipped rect, or an empty rect if they do not intersect.`

Reply:
It shouldn't refer to clipped. It should refer to the intersection between the two rectangles:
> The intersection of this Rect2 and 'b', or an empty Rect2 if they do not intersect.

Reply:
Changed them to `The intersection between the rects, or an empty rect if they do not intersect`, is that good?